src/workspace/src/logger_text/src/logger_text.cpp

```diff
@@ -5,14 +5,37 @@
 #include <cstdio>
 #include <numeric>
 
-Writer::Writer(std::string & outputFolder, std::string separator):outputFolder(outputFolder),separator(separator),transformListener(buffer){
 
+Writer::Writer(std::string & outputFolder, std::string separator):outputFolder(outputFolder),separator(separator),transformListener(buffer){
+			configurationClient = node.serviceClient<setting_msg::ConfigurationService>("get_configuration");
+			ROS_INFO("getting speed threshold configuration...");
+			getSpeedThresholdConfig();
 }
 
 Writer::~Writer(){
 	finalize();
 }
 
+void Writer::getSpeedThresholdConfig(){
```

Comment:
The intent is good but the name could clash with a common interpretation. Don't use an accessor syntax (getXXX) for a void function. Use something more meaningful such as updateSpeedThreshold()

Reply:
fix